src/components/Navbar.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
- [ ] [OPTIONAL] This is great adding snapshot tests for your components, However, I think it could be much better if you have all your tests file in a single place, instead of mixing everything together with your react components, this will help others and even yourself finding quickly all your test files 👍